apps/src/aichat/types.ts

```diff
@@ -3,24 +3,48 @@ import {LevelProperties} from '@cdo/apps/lab2/types';
 // TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
 export type AichatInteractionStatusValue = string;
 
-export type ChatCompletionMessage = {
-  id: number;
-  role: Role;
+export interface ChatItem {
+  // UTC timestamp in milliseconds
+  timestamp: number;
+}
+
+export interface ChatMessage extends ChatItem {
   chatMessageText: string;
+  role: Role;
   status: AichatInteractionStatusValue;
-  chatMessageSuffix?: ChatMessageSuffix;
-  timestamp?: string;
-};
+}
 
-export type AichatCompletionMessage = Pick<
-  ChatCompletionMessage,
-  'role' | 'chatMessageText' | 'status'
->;
+export interface ModelUpdate extends ChatItem {
+  id: number;
+  updatedField: keyof AiCustomizations;
+  updatedValue: AiCustomizations[keyof AiCustomizations];
+}
 
-type ChatMessageSuffix = {
+export interface Notification extends ChatItem {
```

Comment:
Can we remove certain `Role` types now (ERROR_NOTIFICATION, MODEL_UPDATE)?

Reply:
Yep, done!